nsq-mixin/README.md

```diff
@@ -0,0 +1,156 @@
+# NSQ Mixin
```

Comment:
Very thorough README, this should be the standard for all of our mixins. I now have something to aspire to ;)